src/lib/customPropTypes.tsx

```diff
@@ -189,7 +193,7 @@ export const some = validators => (props, propName, componentName, ...rest) => {
   }
 
   const errors = _.compact(
-    _.map(validators, validator => {
+    map(validators, validator => {
```

Comment:
Let's not import `map` alone being that we're already importing all of `lodash/fp` and this file only uses lodash/fp. We should update this call use lodash fp again and swap the iterator with the data:

```jsx
_.map(validator => {}, validators)
```

One of the main differences with `fp` is that it follows the "data last" functional pattern, so the iterator function is the first argument and the data (array in this case) is the last argument.

Reply:
done, thanks